package.json

```diff
@@ -51,6 +51,7 @@
     "eslint": "4.8.0",
     "eslint-config-airbnb-base": "12.0.2",
     "eslint-plugin-import": "2.7.0",
+    "eslint-plugin-mocha": "^4.11.0",
```

Comment:
can you use a fixed version for this in the next PR? i'll merge this now

Reply:
@bd82 don't forget to run `yarn` to regenerate the yarn.lock file!

Reply:
Opps, I used npm install instead of yarn add :(